package.json

```diff
@@ -74,6 +74,68 @@
         "select2": "3.5.1",
         "underscore": "~1.13.6",
         "uuidjs": "^3.3.0",
+        "vue": "^3.3.4",
```

Comment:
Aha! I suppose it could just be that this is the change that has to be merged to 7.5.x before a project can depend on this?

I manually added this to the test project's packages.json and got over the hump 👍